src/ui/webbrowser/webbrowser.ts

```diff
@@ -49,39 +18,33 @@ export declare class AbstractWebBrowserOptions {
  *
  *       @example
  *       import WebBrowser from '@smartface/native/ui/webbrowser';
- *       var webOptions = new WebBrowser.Options();
- *       webOptions.url = "https://smartface.io"
- *       webOptions.barColor = Color.RED;
- *       webOptions.ios.itemColor = Color.BLUE;
- *       WebBrowser.show(page,webOptions);
+ *       const webBrowser = new WebBrowser({
+ *          url: 'https://smartface.io',
+ *          barColor: Color.RED,
+ *          ios: {
+ *           itemColor: Color.GREEN
+ *         }
+ *       });
+ *       webBrowser.show(this);
  */
-export declare class AbstractWebBrowser {
-  /**
-   * @class UI.WebBrowser.Options
-   * @since 2.0.11
-   *
-   * An object that provides a standard interface for browsing the web.
-   *
-   *      @example
-   *      var webOptions = new WebBrowser.Options();
-   *      webOptions.url = "https://smartface.io"
-   *      webOptions.barColor = Color.RED;
-   *      webOptions.ios.itemColor = Color.BLUE;
-   *      WebBrowser.show(page,options);
-   */
-  public static readonly Options: new (params?: Partial<AbstractWebBrowserOptions>) => AbstractWebBrowserOptions;
+export class WebBrowserBase extends NativeComponent {
+  protected createNativeObject(params?: Partial<Record<string, any>>) {
+    throw new Error('createNativeObject Method not implemented.');
+  }
   /**
    *
    * This function shows WebBrowser on the given UI.Page.
    *
    * @event show
    * @param {UI.Page} page
-   * @param {UI.WebBrowser.Options} options
-   * @static
    * @android
    * @ios
    * @since 2.0.11
    */
-  static show(page: Page, options: AbstractWebBrowserOptions): void;
-  constructor(params?: AbstractWebBrowserOptions);
+  show(page: Page) {
+    throw new Error('show() Method not implemented.');
+  }
+  constructor(params?: Partial<IWebBrowserOptions>) {
```

Comment:
I believe the usage will not take constructor parameters since it uses `show` to get options. Therefore we might not take constructor parameters at all.